src/main/java/com/zendesk/maxwell/schema/SchemaPosition.java

```diff
@@ -85,7 +87,7 @@ private void store(BinlogPosition newPosition) {
 		if ( newPosition == null )
 			return;
 
-		String sql = "INSERT INTO `maxwell`.`positions` set "
+		String sql = "INSERT INTO `" + this.schemaDatabaseName + "`.`positions` set "
```

Comment:
either way is fine.  `setCatalog` may be easier from a standpoint of not having to do two connections (one for the `CREATE DATABASE`, the other for the usage)